### PR TITLE
Debug CRA Tests

### LINKED ISSUE #112: Get test debugging working
I believe this requires upgrading node, and possibly `react-scripts`. We should also upgrade the `docs/CreateReactApp.md` docs for the new react scripts on how to do debugging.

### PR DESCRIPTION
**Pull request by fredmon3**
This PR closes #112 

#### What does this PR do?

Adds CRA test debugging.

### CODE REVIEW

**Comment by wdoug** [comment]
@fredmon3 I get the same errors when trying to run this that travis ci is failing with:
```sh
$ npm test
> members@0.1.0 test /home/travis/build/codefordenver/members
> react-scripts test --env=jsdom
 FAIL  src/App.test.js
  ● Test suite failed to run
    /home/travis/build/codefordenver/members/node_modules/graphql/language/printer.mjs:8
    import { visit } from './visitor';
    ^^^^^^
    
    SyntaxError: Unexpected token import
      
      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
      at node_modules/apollo-link/lib/bundle.umd.js:2:141
      at Object.<anonymous> (node_modules/apollo-link/lib/bundle.umd.js:5:2)
Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.248s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

**Comment by wdoug** [comment]
I went ahead and also updated the Create React App docs for this PR as we update `react-scripts`

**Comment by wdoug** [comment]
@fredmon3 this PR now only has the updated docs. The `.vscode` changes have disappeared for some reason

**Review by wdoug — CHANGES_REQUESTED** [review]
Let's add back the `.vscode` stuff for this

**Comment by fredmon3** [comment]
@wdoug How's it looking?

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+      {
+        "name": "Chrome",
```
Comment (wdoug) [inline .vscode/launch.json:8]: What does the chrome debugger do?

Comment (wdoug) [reply .vscode/launch.json:8]: Nevermind, I see in the `CreateReactApp.md` document

**Review by wdoug — APPROVED** [review]
Looks good. It might be nice to include a note in the readme how to use the vscode debugger. Maybe just copying some of the text from the CreateReactApp.md file, or at least linking to it

**Pull request merged**